sdk/formrecognizer/Azure.AI.FormRecognizer/tests/DocumentAnalysisClient/DocumentAnalysisClientMockTests.cs

```diff
@@ -359,6 +467,184 @@ public async Task AnalyzeDocumentCanParseDocumentStyleWithV410Properties()
             Assert.AreEqual("backgroundColorValue", style.BackgroundColor);
         }
 
+        [Test]
+        public async Task AnalyzeDocumentCanParseDocumentAnnotation()
```

Comment:
We do not have test files we can make public to test any of these features, so I'm adding mock tests for now. They have been live tested locally, though, and all of them work.